File: Day 1 Data Types.py
import sys


if len(sys.argv) != 4:
    print("must enter 3 values")
    exit(1)

cost = float(sys.argv[1])
tip = float(sys.argv[2])
tax = float(sys.argv[3])

tipPercent = (cost * tip) / 100
taxPercent = (cost * tax) / 100
totalCost = cost + tipPercent + taxPercent
l = totalCost
m = (totalCost * 100) - (1 * 100)
if m >= 50:
    l = l + 1
print(f'The total meal cost is {round(l,2)} dollars.')

File: Day 1/Day 1 Data Types.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
         float mealcost,tippercent,taxpercent,totalcost;
    int tip,tax,l,m;
    scanf("%f\n%d\n%d",&mealcost,&tip,&tax);
    tippercent=mealcost*tip/100;
    taxpercent=mealcost*tax/100;
    totalcost=mealcost+tippercent+taxpercent;
    l=totalcost;
    m=(totalcost*100)-(l*100);
    l=(m>=50)?l+1:l;
    printf("The total meal cost is %d dollars.",l);
    return 0;
}
